src/ContentHubClient.php

```diff
@@ -1028,10 +1028,7 @@ public function unSuppressWebhook(string $webhook_uuid) {
   public function deleteClient($client_uuid = NULL) {
     $settings = $this->getSettings();
     $uuid = $client_uuid ?? $settings->getUuid();
-    $response = $this->deleteEntity($uuid);
-    if (!$response || $response->getStatusCode() < 200 || $response->getStatusCode() >= 300) {
-      throw new \Exception(sprintf("Entity with UUID = %s cannot be deleted.", $uuid));
-    }
+    $this->deleteEntity($uuid);
```

Comment:
In this case we wouldn't know why we were not able to delete the client CDF if there's no indication. The exception was needed I guess to check what the reason was, because we cannot return multiple responses. The only case basically when it is fine if there wasn't any client, when the service responds with a 404. Thoughts?